disperser/dataapi/server.go

```diff
@@ -165,7 +166,7 @@ func (s *server) Start() error {
 			metrics.GET("/throughput", s.FetchMetricsTroughputHandler)
 			metrics.GET("/non_signers", s.FetchNonSigners)
 			metrics.GET("/operator_nonsigning_percentage", s.FetchOperatorsNonsigningPercentageHandler)
-			metrics.GET("/deregistered_operators", s.FetchDeregisteredOperators)
+			metrics.GET("/deregistered-operators", s.FetchDeregisteredOperators)
```

Comment:
Why is there `/operators-info/deregistered-operators` and  `metrics/deregistered-operators`?